### PR TITLE
Update link to examples

### PR DESCRIPTION
**Pull request by Bachibouzouk**
- Remove the old example
- Add a link to the examples in place of the removed file
- Update the link in the README

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -20,12 +20,15 @@ Here is a template for new release sections
...
 ## [Unreleased]
 
-
-## [0.0.7] 2019-08-29
-
 ### Added
 - CHANGELOG
 
+### Changed
+- updated link to the examples/tutorials ([#24](https://github.com/OpenEnergyPlatform/oedialect/pull/24)
```
Comment (nesnoj) [inline CHANGELOG.md]: Missing closing parenthesis

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,12 +20,15 @@ Here is a template for new release sections
...
+
+### Removed
+- notebook example ([#24](https://github.com/OpenEnergyPlatform/oedialect/pull/24)
```
Comment (nesnoj) [inline CHANGELOG.md]: dito

File: README.md [hunk 3]
```diff
@@ -12,6 +12,7 @@ This repository is licensed under [GNU Affero General Public License v3.0 (AGPL-
 
 pip install oedialect
 
-## Example
+## Tutorials
 
-You can find a basic example [here](doc/example/oedialect_basic_example.ipynb).
+You can find tutorials and examples [here](https://github
```
Comment (nesnoj) [inline README.md]: Link broken (due to line break)

**Comment by Bachibouzouk** [comment]
> Why did you delete the simple example? Is it obsolete / included in the other tutorials?

@nesnoj I assumed it was present in the example repo. I plan to reorganize the example repo, I'll add the file there if it did not exist :)

Thanks for your review :)

File: CHANGELOG.md (hunk 1 above)
Comment (Bachibouzouk) [reply CHANGELOG.md]: Fixed in b301e656cf4a0a4e2ce87fe05480e645c9f69253

File: CHANGELOG.md (hunk 2 above)
Comment (Bachibouzouk) [reply CHANGELOG.md]: Fixed in b301e656cf4a0a4e2ce87fe05480e645c9f69253

File: README.md (hunk 3 above)
Comment (Bachibouzouk) [reply README.md]: Fixed in 9b9108b0faacb4bbc47f70674cd722737aecc01b

**Comment by MGlauer** [comment]
> > Why did you delete the simple example? Is it obsolete / included in the other tutorials?
> 
> @nesnoj I assumed it was present in the example repo. I plan to reorganize the example repo, I'll add the file there if it did not exist :)
> 
> Thanks for your review :)

I am a bit conflicted regarding the location of the examples. I would like to keep them with the code (i.e. inside the documentation) so we can implement proper doctests with travis etc.
The problem is examples that work directly on the OEP (e.g. via hard-coded urls). It might be a bad idea to include those in automated tests.

**Comment by Bachibouzouk** [comment]
> so we can implement proper doctests with travis etc.

We can perfeclty implement doctests in the examples repository, no? As the repository [examples](https://github.com/OpenEnergyPlatform/examples) contained already 5 files which explicitly had `oedialect in their names` and because of issue #9 I thought it would be simpler to move the example where the majority of the file lies already.

If we decide the examples should not be in the repository `examples`, then we might also consider moving the examples back into their own repos (i.e. oedialect, oeplatform) and get rid of example. I would be more inclined to keep examples now that it was created.

**Pull request merged**